website/docs/quickstarts/bigquery-qs.md

```diff
@@ -100,13 +100,10 @@ Now that you have a repository configured, you can initialize your project and s
 
 1. Click **Start developing in the IDE**. It might take a few minutes for your project to spin up for the first time as it establishes your git connection, clones your repo, and tests the connection to the warehouse.
 2. Above the file tree to the left, click **Initialize dbt project**. This builds out your folder structure with example models.
-3. Make your initial commit by clicking **Commit and sync**. Use the commit message `initial commit` and click **Commit**. This creates the first commit to your managed repo and allows you to open a branch where you can add new dbt code.
+3. Make your initial commit by clicking **Commit and sync**. Use the commit message `initial commit` and click **Commit**. This creates the first commit to your managed repo and allows you to open a branch where you can add a new dbt code.
 4. You can now directly query data from your warehouse and execute `dbt run`. You can try this out now:
-    - Click **+ Create new file**, add this query to the new file, and click **Save as** to save the new file:  
```

Comment:
Addressing the issue.

Reply:
@anksu2024 Removing this might muddle the instructions. Would it be better to direct the customer to put this on a different branch?

Reply:
I followed the instructions and the same thing we are doing in the next section. i.e. we create a new branch and our first model on the branch. Hence, it seems like, the learner's flow will become more streamlined. 

As a learner myself, I struggle for some time with this. Hence, thought of adding.

Reply:
@matthewshaver : Following up on this. Thx.

Reply:
Nevermind! I see you added it, Thank you!